### ISSUE TITLE
iGoat.ipa is not a valid IPA

### ISSUE THREAD

**Issue by imnarendrabhati** [OWASP/igoat#42]
My Environment 
iOS 9.3.3 Jailbroken
Device - iPad Air 2

Downloaded the IPA file
https://github.com/OWASP/igoat/blob/master/iGoat.ipa

Installing using "installipa iGoat.ipa " But its showing the error.

"private/var/tmp/iGoat.ipa is not a valid IPA" snap shot attached
![ipa error](https://user-images.githubusercontent.com/3678330/30844648-88a60dd0-a2ad-11e7-9a73-4b47b76b56bc.JPG)

**Comment by imnarendrabhati** [comment]
iFunBox is also unable to install the application. 
![ifun box](https://user-images.githubusercontent.com/3678330/30845036-38c33dea-a2af-11e7-82ea-c16c4259debb.JPG)

**Comment by swaroopsy** [comment]
![ipa_installation](https://user-images.githubusercontent.com/9929997/31353180-253ee1d4-ad4f-11e7-955d-f5ebc4889c59.png)
Hi @imnarendrabhati I just re-rechecked and it's working fine. Make sure you've installed appsync unified along with installipa. I've attached screenshot of working ipa on iOS 9.3.3.

**Comment by swaroopsy** [comment]
Hi @imnarendrabhati I just re-rechecked and it's working fine. Make sure you've installed appsync unified along with installipa. I've attached screenshot of working ipa on iOS 9.3.3.
![ipa_installation](https://user-images.githubusercontent.com/9929997/31353263-78da374e-ad4f-11e7-8908-c667fa001892.png)

**Comment by pig837** [comment]
It is a slightly different situation from me, I failed to install iGoat 3.0 Package with error message. (iPhone 6 Plus, iOS 10.3.3 with DoubleH3lix RC6)

It's not working with below tools, also.
ipainstaller, AppSync Unified 4.0, iFunBox 4.0.4105.1352(x64), Filza 3.4

Thus I converted ipa to deb file, eventually I successfully installed iGoat 3.0.
[Reference] https://k2designlab.wordpress.com/2015/05/13/convert-ipa-file-into-deb-file-for-cydia-repo-distribution/
